modules/elasticsearch/README.md

```diff
@@ -0,0 +1,96 @@
+# AWS Elasticsearch module
+
+This modules creates an Elasticsearch cluster in a domain, with redirection
+service to allow easy access to Kibana web interface (requires DNS CNAME alias
+to be set up).
+
+## Registered `consul` service name
+
+The registered `consul` service name is `elasticsearch`, and the default port
+used is `443`.
+
+The actual VPC service and port are registered in `consul`. Any other services
+that require Elasticsearch service should always use the actual VPC service
+name, since the service is hosted under SSL and the SSL certificate to accept is
+registered under the VPC name (and not the `consul` service name).
+
+## Example Terraform configuration
```

Comment:
Not sure how useful the example is when most of it is `var.xxx`.  We should tweak this to be some "quick start" example instead.

- Show how we can use the outputs from `core` module using `data.terraform_remote_state`by having the data resource in the example (and `traefik` if they want to use the redirect job)
- Only show the "required" variables and some example values